.github/workflows/flutter_package.yml

```diff
@@ -30,7 +30,6 @@ on:
       format_line_length:
         required: false
         type: string
-        default: "80"
```

Comment:
I think we should keep the default, based on #329.

Since we do not have the:

> If we want to be super backward compatibility with older darts, we can also add a checking on the dart version and if it is lower than 3.7, the flag continues to be explicitly added

I think having the default, although redundant ensures we support older Dart versions. Thoughts?